scripts/st-ctl.sh

```diff
@@ -213,9 +213,11 @@ if $cmd_set_concord_log_level; then
         "concord.bft.st.dst" \
         "concord.bft.st.src" \
         "concord.util.handoff" \
+        "concord.bft.st.dbdatastore" \
+        "concord.bft.st.inmem" \
         # Uncomment as needed
         #"serializable" \
-        #"concord.bft.st.dbdatastore" \
+        #"concord.bft.st.inmem" \
```

Comment:
Is this needed? It is already in use as can be seen a few lines above.

Reply:
Thanks, i'll remove it.